test-app/package.json

```diff
@@ -54,7 +54,7 @@
     "ember-page-title": "8.0.0-beta.0",
     "ember-qunit": "^5.1.5",
     "ember-resolver": "^8.0.3",
-    "ember-source": "~3.28.3",
+    "ember-source": "~3.20.5",
```

Comment:
Was this change intended?

Reply:
indeed no! I'm not sure how it happened 🤦